content/en/docs/reference/glossary/security-context.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Security Context
+title: Security context
```

Comment:
why? Probably had this conversation before, but I _think_ the glossary terms capitalize the words in the title

Reply:
Ah yeah I remember now!